include/mbedtls/mbedtls_config.h

```diff
@@ -1364,6 +1364,15 @@
  */
 //#define MBEDTLS_PSA_CRYPTO_CLIENT
 
+/**
+ * \def MBEDTLS_PSA_ENABLE_KEY_ENROLLMENT
```

Comment:
How much code size will this save (once it's finished)?

Reply:
For this question I replied in the [comment below](https://github.com/Mbed-TLS/mbedtls/pull/8449#issuecomment-1790233521)

Reply:
This is not about code size. This is about compliance with the PSA crypto APIs implemented as services e.g. by TF-M. alg2 and the key enrollment concept doesn't exist in TF-M.

Reply:
The existence of this feature doesn't break compliance with the PSA crypto API, and services built on top of Mbed TLS don't need to support setting the enrollment algorithm.

Can you please describe the problem you're facing? If it's not code size then I suspect you're trying to fix it in the wrong place.